src/main/flight/pid.c

```diff
@@ -204,7 +204,7 @@ void resetPidProfile(pidProfile_t *pidProfile)
         .dyn_idle_max_increase = 150,
         .feedforward_averaging = FEEDFORWARD_AVERAGING_OFF,
         .feedforward_max_rate_limit = 100,
-        .feedforward_smooth_factor = 37,
+        .feedforward_smooth_factor = 25,
```

Comment:
changing default here because ?

Reply:
Second order filtering on the 'boost' element is now more effective than before.  We can now get equivalent overall smoothing with a lower smooth factor.
250hz RC links have a signal to noise ratio roughly half that of 'normal' FrSky or Crossfire links, on which these defaults are based.  Typically they require 2 point averaging and smoothing of 33-40.  
500hz RC links have even lower signal to noise ratio, and will require 2 point averaging and smoothing of 50 or higher.
With higher rate RC links, the optimal settings depend greatly on gimbal quality and other factors.